### PR TITLE
Project copy ready for review.

### PR DESCRIPTION
**Pull request by Vgonma**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+function populateList(emptyList) {
+  emptyList.sortTasks();
+  for (let i = 0; i < emptyList.taskList.length; i += 1) {
+    const temp = createTaskDOM(emptyList.taskList[i].description);
+    taskList.appendChild(temp);
+    addCheckBoxListener(temp.children[0]);
+  }
+}
```
Comment (gjuliao) [inline src/index.js]: This might nice to try with a forEach

File: src/modules/ToDoList.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    }
+  }
+
+  removeCompleted() {
+    this.taskList = this.taskList.filter((task) => !task.completed);
+  }
+
+  sortTasks() {
+    for (let i = 0; i < this.taskList.length - 1; i += 1) {
+      for (let j = 0; j < (this.taskList.length - 1) - i; j += 1) {
```
Comment (gjuliao) [inline src/modules/ToDoList.js:42]: It might be good to check if, with the sort method this could work much cleaner!

**Review by gjuliao — CHANGES_REQUESTED** [review]
# Request changes ♻️ 
You have done a great job so far! 🥳  Still minor details to work on.

Please review my comments below and request a review as soon as you are done!

Please let me know if you have any comments!

Happy coding! 👨‍💻

File: src/modules/ToDoList.js (hunk 2 above)
Comment (Vgonma) [reply src/modules/ToDoList.js:42]: Thanks! I agree it looks kind of ugly but that is the standard way of doing the bubble sort which is what I used.

File: src/index.js (hunk 1 above)
Comment (Vgonma) [reply src/index.js]: Thanks that is true. It will be easier to read. 👍

**Review by gjuliao — APPROVED** [review]
Great Job Victor! 🎉 🎉 

You can take a break now! 💯

**Pull request merged**